sdk/eventhub/Azure.Messaging.EventHubs/src/Authorization/SharedAccessSignature.cs

```diff
@@ -149,14 +149,14 @@ public SharedAccessSignature(string sharedAccessSignature) : this(sharedAccessSi
         }
 
         /// <summary>
-        ///   nitializes a new instance of the <see cref="SharedAccessSignature" /> class.
+        ///   Initializes a new instance of the <see cref="SharedAccessSignature" /> class.
         /// </summary>
         ///
         /// <param name="eventHubResource">The Event Hubs resource to which the token is intended to serve as authorization.</param>
         /// <param name="sharedAccessKeyName">The name of the shared access key that the signature should be based on.</param>
         /// <param name="sharedAccessKey">The value of the shared access key for the signagure.</param>
         /// <param name="value">The shared access signature to be used for authorization.</param>
-        /// <param name="expirationUtc">The date and time that the shared access signature expires, in UTC.</param>
+        /// <param name="expirationTime">The date and time that the shared access signature expires.</param>
```

Comment:
I'm not sure that `xxxTime` is the pattern that we want to use.   Though, `expiration` or `expirationDateTime` doesn't fit.   Maybe `signatureExpiration` or something else contextually relevant would be more descriptive?

Reply:
Renamed it to `signatureExpiration`.